src/main/java/org/apache/commons/imaging/formats/jpeg/decoder/JpegDecoder.java

```diff
@@ -46,6 +46,10 @@
 import org.apache.commons.imaging.formats.jpeg.segments.SosSegment;
 
 public class JpegDecoder extends BinaryFileParser implements JpegUtils.Visitor {
+
+    private static final int [] BAND_MASK_ARGB = {0x00ff0000, 0x0000ff00, 0x000000ff, 0xff000000};
```

Comment:
OK, you define the constants but do not use them. What am I missing?

Reply:
Oops.  I used them on lines 468 and 472.  But I neglected to remove the old lines on 467 and 471.  Will submit and additional change.